### DISCUSSION TITLE
I built something similar a couple of years ago

### DISCUSSION THREAD

**Discussion by koona-labs** [ismaelw/laratex#23] [Show and tell]
Hey there, 

first of all: Thanks for this nice package. It looks awesome. 

I remember that I built a similar package a couple of years ago. There I built an analogous api to Mailables that keep all the logic (like additional assets, how many times a document is compiled). 

It was published under the account of my former company. If you like to check it out, you'll find it under https://github.com/abiturma/laravel-latex

Best regards
KoonaLabs

**Comment by ismaelw** [comment]
Hi @koona-labs 

Thank you so much for your feedback.
I was just looking into your package and I really like the LatexEscaper and the Blade directive "@latex". Smart idea which is definitely missing in my package.

Do you think there is a chance to make a collaboration to add the feature into this package?

I can see multiple usecases where this would be beneficial in my projects where I use my LaraTeX package.

Kind regards,
Ismael

**Reply by koona-labs** [reply]
Hey Ismael, 

thanks for your nice reply. Yeah, of course I see a chance of collaboration. Either I'd  just try to contribute in form of a PR or you can just feel free to copy functionalities over to your project. In the latter case I'm happy about a brief mention/credit of my work, but it's no must of course.

Best regards
Aaron

**Reply by ismaelw** [reply]
Hi Aaron,

If you want you can let me know if you find the time soon. I guess I would have to get deeper into your code first to know what's going on there. And I'm not sure when I can sit down to do that. 

I definitely think it's the right next feature to implement.

Kind regards,
Ismael

**Reply by koona-labs** [reply]
Hey Ismael, 

no problem. I'm also quite busy these days. But the "@latex" directive is really easy. I sent you a PR for it. 

Best regards
Aaron

**Reply by ismaelw** [reply]
Hey Aaron,

Thank you so much. I tested it really quick locally and it seems to be working :)
I merged your PR and created a new release.

Thank you very much for your contribution!

Kind regards,
Ismael
